specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2023-07-01/imagebuilder.json

```diff
@@ -0,0 +1,2041 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "VirtualMachineImageTemplate",
+    "description": "Virtual Machine Image Template",
+    "version": "2023-07-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VirtualMachineImages/imageTemplates": {
+      "get": {
+        "tags": [
+          "VirtualMachineImageTemplate"
+        ],
+        "operationId": "VirtualMachineImageTemplates_List",
+        "description": "Gets information about the VM image templates associated with the subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ImageTemplateListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
```

Comment:
It should be "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"

Reply:
fixed